apps/api/src/app/workflows/usecases/update-notification-template/update-notification-template.usecase.ts

```diff
@@ -349,6 +363,65 @@ export class UpdateNotificationTemplate {
     return removedStepsIds;
   }
 
+  private async updateVariants(
+    message: NotificationStep,
+    command: UpdateNotificationTemplateCommand,
+    parentChangeId: string,
+    parentStepId: string | null
+  ): Promise<StepVariantEntity[]> {
+    if (!message.variants?.length) return [];
+
+    const variantsList: StepVariantEntity[] = [];
+    let parentVariantId: string | null = null;
+
+    for (const variant of message.variants) {
+      if (!variant.template) throw new ApiException(`Unexpected error: variants message template is missing`);
+
+      const variantTemplate = await this.createMessageTemplate.execute(
```

Comment:
It is possible that he is updating an existing variant, so it won't always be the createMessageTemplate @djabarovgeorge

Reply:
Will add the logic we have on the step update flow.